Item12225: Don't use "HEAD" to detect pseudo install.
    
    A real, non-pseudo-installed extension will crash configure if a perl
    version object is compared to an alpha string.
    
    9999.99_999 will be used to indicate a pseudo-installed release.

~~~PATCH 4e525d29a245785ae810316835c18e62:70a6a0fd9f6a377f212beafc2e5758cd lib/Foswiki/Configure/Dependency.pm (Foswiki-1.1.4)
diff --git lib/Foswiki/Configure/Dependency.pm lib/Foswiki/Configure/Dependency.pm
--- lib/Foswiki/Configure/Dependency.pm
+++ lib/Foswiki/Configure/Dependency.pm
@@ -220,7 +220,7 @@ sub studyInstallation {
             if ( -l "$dir/$path" ) {
 
                 # Assume pseudo-installed
-                $this->{installedVersion} = 'HEAD';
+                $this->{installedVersion} = '9999.99_999';
             }
             last;
         }
@@ -459,6 +459,7 @@
         #print STDERR "Expecting SVN comparison, but RELEASE was $baseType \n";
         ( $baseType, @atuple ) = _decodeReleaseString($aVERSION)
           if ( defined $aVERSION && $baseType ne 'svn' );
+        return 1 if ( $baseType eq 'tuple' );
         return 0 unless ( $baseType eq 'svn' );
 
     }

~~~PATCH 76e28354522a6d6cccc76c66f99d2424:24ce3ef562c0021782b103c81774b980 lib/Foswiki/Configure/UIs/EXTENSIONS.pm (Foswiki-1.1.4)
diff --git lib/Foswiki/Configure/UIs/EXTENSIONS.pm lib/Foswiki/Configure/UIs/EXTENSIONS.pm
--- lib/Foswiki/Configure/UIs/EXTENSIONS.pm
+++ lib/Foswiki/Configure/UIs/EXTENSIONS.pm
@@ -280,7 +280,7 @@ sub getExtensions {
         if ( $ext->{installedRelease} ) {
 
             # The module is installed; check the version
-            if ( $ext->{installedVersion} eq 'HEAD' ) {
+            if ( $ext->{installedVersion} eq '9999.99_999' ) {
 
                 # pseudo-installed
                 $install = 'pseudo-installed';

